rpm_spec/subpackages/manageiq-gemset

```diff
@@ -25,6 +25,9 @@ Requires: cyrus-sasl
 Requires: cyrus-sasl-plain
 Requires: qpid-proton-c
 
+# For RHV
+Requires: ovirt-ansible-roles
+
```

Comment:
Do we also need `python3-ovirt-engine-sdk4`?

Reply:
ovirt-ansible-roles requires python3-ovirt-engine-sdk4, so no need to explicitly add. Sorry, I should have mentioned in the PR description.